installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc

```diff
@@ -27,9 +27,8 @@ The {product-title} installation program offers the following methods for deploy
 
 * *Interactive*: You can deploy a cluster with the web-based link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform[{ai-full}]. This method requires no setup for the installer, and is ideal for connected environments like {ibm-z-name}.
 
-* *Local Agent-based*: You can deploy a cluster locally with the xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#preparing-to-install-with-agent-based-installer[Agent-based Installer]. It provides many of the benefits of the {ai-full}, but you must download and configure the Agent-based Installer first. Configuration is done with a command line interface (CLI).
+* *Local Agent-based*: You can deploy a cluster locally with the xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#preparing-to-install-with-agent-based-installer[Agent-based Installer]. It provides many of the benefits of the {ai-full}, but you must download and configure the Agent-based Installer first. Configuration is done with a command line interface (CLI). This approach is ideal for air-gapped or restricted networks.
```

Comment:
https://redhat-documentation.github.io/supplementary-style-guide/#disconnected 
I did see "air-gapped" being used for bare metal and VSphere. So feel free to ignore my comment if "air-gapped" suits best here.

Reply:
Thanks for sharing I didn't know that but I like the term "disconnected" most. Will change

Reply:
@mjpytlak @bscott-rh Is that something we should look into? I think in most of the install docs we use restricted.

Reply:
There has been discussion over time about whether "restricted" or "disconnected" is the preferred term for this. I'm not aware of any real conclusion from those discussions.